addons/binding/org.openhab.binding.pulseaudio/src/main/java/org/openhab/binding/pulseaudio/handler/PulseaudioBridgeHandler.java

```diff
@@ -142,8 +142,8 @@ public void run() {
                             startAutomaticRefresh();
                         }
                     } catch (IOException e) {
-                        logger.error("Couldn't connect to Pulsaudio server [Host '" + host + "':'" + port + "']: "
-                                + e.getLocalizedMessage());
+                        logger.error("Couldn't connect to Pulsaudio server [Host '{}':'{}']: {}", host, port,
+                                e.getLocalizedMessage());
```

Comment:
I would log the exception instead, it is an error, I want to be able to debug the stacktrace